docs/content/docs/features/primary-key-table.md

```diff
@@ -158,12 +154,21 @@ Full compaction changelog producer can produce complete changelog for any type o
 
 {{< /hint >}}
 
-## Changelog Tables without Primary Keys
+## Sequence Field
```

Comment:
Change this as a sub-section of "Merge Engines", because it only affects the result of deduplicate merge engine.

Reply:
I think it is also related to the changelog, it will affect the changelog result.